types/types.go

```diff
@@ -186,7 +186,9 @@ func WithAttachments(rootAppDir string) func(*App) error {
 			if err != nil {
 				return err
 			}
-
+			if strings.HasPrefix(path, filepath.Join(rootAppDir, "images")) {
```

Comment:
Would be good to add a test for this case

Reply:
What about 
```go
if path == filepath.Join(rootAppDir, "images") {
    return filepath.SkipDir
}
``` ?